Incorporate User Resources/Readme.md

```diff
@@ -1,59 +1,78 @@
 
 # Dependencies
 - LabVIEW NXG 2.0 Beta
 
 # Setup
-- Clone the [ni/webvi-examples](https://github.com/ni/webvi-examples) repo to your machine.
-- Open Incorporate User Resources/Incorporate User Resources.lvproject
-- Open WebApp.gcomp, and then Main.gviweb in that component.
+1. Clone the [ni/webvi-examples](https://github.com/ni/webvi-examples) repository to your machine.
+2. Open `Incorporate User Resources/Incorporate User Resources.lvproject`.
+3. On the **Project Files** tab, expand `WebApp.gcomp`.
+4. Double-click `Main.gviweb` to open it.
 
 # Details
-## Example Information
-The included example uses two types of resources: images, and CSS files (stylesheets).
+This example uses two types of resources: images and CSS files.
+
+Two images are included in a child namespace within the `WebApp.gcomp` file called `images`: the National Instruments logo (`.svg`), and a screenshot of the diagram code (`.png`).
 
-2 images are included in a child namespace called 'images': the National Instruments logo (.svg), and a screenshot of the block diagram code (.png). A CSS file is also included (styles/main.css), which styles the Web VI header at the top of the panel. See the next section for information on how these resources are referenced from the panel.
+A CSS file is included in a child namespace called `styles`. The file `main.css` styles the WebVI header at the top of the panel. 
 
-## Using User Resources in LabVIEW NXG
-The Web VI has several ways to reference existing content (resource files). These resources can be added from the LabVIEW NXG, viewed from the editor at edit-time and run-time, and will also be included in a built Web Application Component.
+Refer to the following sections to learn how to add resource files to a web application component and reference them in a WebVI.
 
-To add resource files:
-- Open the Web Application Component (.gcomp) document
-- Decide on a namespace / folder hierarchy for your files. Files can be added to the root namespace, but you may want to create child namespaces / folders to put them in, such as a "styles" folder for CSS files, an "images" for image files, etc.
-- Select the destination namespace in the component document, and choose "Import File(s)". Note that you must have already saved your Web Application Component to disk for the import option to be enabled.
+### Adding resource files to a web application component
+1. Open or create a web application project.
+2. On the **Project Files** tab, double-click `WebApp.gcomp` to open the web application component.
+3. Decide on a folder structure for your files. You can add all files to the same folder, or you can organize files in subfolders, such as a `styles` folder for CSS files or an `images` folder for image files.
+4. _(Optional)_ Create namespaces that correspond to each folder in your folder structure.
```

Comment:
Missing newlines for lines 28-30. Probably worth checking the rest of the docs (I commented on some but there may be more). Anything using the 'a.' 'b.' sub-bulleting without 2 spaces at the end of the line, gets rendered as a single line in Markdown